doc/rtd/topics/datasources/ec2.rst

```diff
@@ -80,6 +81,15 @@ The settings that may be configured are:
  * **timeout**: the timeout value provided to urlopen for each individual http
    request.  This is used both when selecting a metadata_url and when crawling
    the metadata service. (default: 50)
+ * **apply_network_config**: Boolean (default: True) to allow cloud-init
```

Comment:
It's clearer here, but I still think this name is too broad. If False, we will still apply network configuration to the system (although of course it will be a different configuration). (You probably have a better sense of what the scope of this variable is, so I won't suggest anything.)

Reply:
yeah, I was compelled to align with both OpenStack and Azure config options trying to keep each comparable solution with the same name. Maybe none of them are appropriate but they are probably all about equally bad